telos-frontend/src/components/todo/JournalTodo.jsx

```diff
@@ -269,7 +259,8 @@ const JournalTodo = ({ data, date }) => {
             label="Disabled"
             endAdornment={
               <InputAdornment position="end">
-                <IconButton
+              {/* When the user clicks on the button a modal will pop up allowing the user to set their requirements */}
```

Comment:
Same here

Reply:
I have aligned the code with the comment required